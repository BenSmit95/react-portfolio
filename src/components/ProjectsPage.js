import React from 'react';

const ProjectsPage = () => (
    <div>
        ProjectsPage component.
    </div>
)

export default ProjectsPage;