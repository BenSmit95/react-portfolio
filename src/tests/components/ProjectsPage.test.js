import React from 'react';
import { shallow } from 'enzyme';
import ProjectsPage from '../../components/ProjectsPage';

test('Should render ProjctsPage component correctly', () => {
    const wrapper = shallow(<ProjectsPage />);
    expect(wrapper).toMatchSnapshot();
});