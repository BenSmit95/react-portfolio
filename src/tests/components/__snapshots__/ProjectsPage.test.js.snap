// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ProjctsPage component correctly 1`] = `
<div>
  ProjectsPage component.
</div>
`;
